src/meta/processors/admin/Balancer.cpp

```diff
@@ -580,79 +693,85 @@ int32_t Balancer::acquireLeaders(
                         std::back_inserter(diff));
     auto& hostLeaders = leaderHostParts[host];
     for (const auto& partId : diff) {
+        VLOG(3) << "Try acquire leader for part " << partId;
+        auto minLoad = partBounds_[partId].first;
         // find the leader of partId
         auto peers = peersMap[partId];
         for (const auto& peer : peers) {
             if (peer == host || !activeHosts.count(peer)) {
                 continue;
             }
             // if peer is the leader of partId and can transfer, then transfer it to host
+            // if the space specified with zone, we should check for conflict.
```

Comment:
Where do we check conflict? Is this right?

Reply:
this message has discard